src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -309,6 +309,10 @@ class ContentExplorer extends Component {
         const { items } = this.props;
         const item = items[index];
 
+        if (item.isDisabled || item.isLoading) {
```

Comment:
Can you add a test for this?
Here's some similar tests you can base it on: https://github.com/box/box-ui-elements/blob/master/src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js#L272 & https://github.com/box/box-ui-elements/blob/master/src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js#L388

Reply:
Sure thing, thanks!

Reply:
Added